slave.mk

```diff
@@ -209,6 +209,7 @@ $(info "BLDENV"                          : "$(BLDENV)")
 $(info "VS_PREPARE_MEM"                  : "$(VS_PREPARE_MEM)")
 $(info "ENABLE_SFLOW"                    : "$(ENABLE_SFLOW)")
 $(info "ENABLE_NAT"                      : "$(ENABLE_NAT)")
+$(info "TELEMETRY_READWRITE"             : "$(TELEMETRY_READWRITE)")
```

Comment:
Have you tested this change? 
Setting this flag to Yes, while calling Make, did it propagate to Make of Telemetry?

Reply:
I have tested like this:
`TELEMETRY_READWRITE=y make target/debs/stretch/sonic-telemetry_0.1_amd64.deb` and loaded to a VM switch and confirmed that gnmi_set is allowed.

Then:
`make target/debs/stretch/sonic-telemetry_0.1_amd64.deb` and loaded to VM switch and confirmed that gnmi_set is not allowed with "Unimplemented" and error message: "Telemetry is in read-only mode"